src/ios/IntercomBridge.m

```diff
@@ -9,7 +9,7 @@ + (void)setCordovaVersion:(NSString *)v;
 @implementation IntercomBridge : CDVPlugin
 
 - (void)pluginInitialize {
-    [Intercom setCordovaVersion:@"3.0.11"];
+    [Intercom setCordovaVersion:@"3.0.13"];
```

Comment:
npm is failing on install now:

`npm ERR! fetch failed https://registry.npmjs.org/cordova-plugin-intercom/-/cordova-plugin-intercom-3.0.12.tgz`